ci/buildkite.yml

```diff
@@ -23,6 +23,17 @@ steps:
         limit: 2
     agents:
       - "queue=cuda"
+  - command: "ci/test-large-network.sh"
+    name: "large-network [public]"
+    env:
+      CARGO_TARGET_CACHE_NAME: "stable"
+    timeout_in_minutes: 20
+    retry:
```

Comment:
Do we need the `retry:`?  I'd rather remove it, I added it elsewhere a while ago due to flaky tests (probably should be removed from there eventually too)